src/libraries/icubmod/embObjMultipleFTsensors/CMakeLists.txt

```diff
@@ -9,16 +9,17 @@ yarp_prepare_plugin(embObjMultipleFTsensors CATEGORY device
                                  INCLUDE embObjMultipleFTsensors.h
                                  EXTRA_CONFIG WRAPPER=analogServer)
 
-IF (NOT SKIP_embObjMultipleFTsensors)
+if (NOT SKIP_embObjMultipleFTsensors)
 
 
   set(ICUB_COMPILE_EMBOBJ_LIBRARY ON CACHE INTERNAL "use the embObjLib lib")
 
 
-  INCLUDE_DIRECTORIES(${CMAKE_CURRENT_SOURCE_DIR})
+  include_directories(${CMAKE_CURRENT_SOURCE_DIR})
 
   yarp_add_plugin(embObjMultipleFTsensors embObjMultipleFTsensors.cpp embObjMultipleFTsensors.h)
-  TARGET_LINK_LIBRARIES(embObjMultipleFTsensors ethResources)
+  target_link_libraries(embObjMultipleFTsensors ethResources
+                        YARP::YARP_os YARP::YARP_dev YARP::YARP_sig)
```

Comment:
`embObjMultipleFTsensors` also depends on YARP_os and YARP_sig (including headers).